File: INICIANTE/1074/main.c
#include <stdio.h>

int main() {
    int n, x;
    scanf("%d", &n);
    while (n--) {
        scanf("%d", &x);
        if (x == 0) printf("NULL\n");
        else {
            printf((x & 1) == 0 ? "EVEN ": "ODD ");
            printf(x < 0 ? "NEGATIVE\n": "POSITIVE\n");
        }
    }
    return 0;
}

File: INICIANTE/1074/main.js
let lines = require('fs').readFileSync('/dev/stdin', 'utf-8').split('\n');
let n = parseInt(lines.shift());
while (n--) {
    let x = parseInt(lines.shift());
    if (x === 0) console.log('NULL');
    else {
        process.stdout.write(x % 2 ? 'ODD ' : 'EVEN ');
        console.log(x < 0 ? 'NEGATIVE' : 'POSITIVE');
    }
}
